ui/pages/create-account/connect-hardware/account-list.js

```diff
@@ -73,7 +73,7 @@ class AccountList extends Component {
       HardwareDeviceNames.lattice,
       HardwareDeviceNames.trezor,
       HardwareDeviceNames.oneKey,
-    ].includes(device.toLowerCase());
+    ].some((name) => name === device);
```

Comment:
Since we got rid of the `toLowerCase` we can probably re-use `includes` which is slightly better to read IMO

```suggestion
    ].includes(device);
```